app/posts/govuk-eleventy-plugin.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Write documentation using Markdown and publish it using GOV.UK styles
+description: The GOV.UK Eleventy Plugin makes it easy to start writing documentation rather than spend time building a website for it.
+date: 2022-04-04
+tags: post
+layout: post
+author:
+  name: Paul Lloyd
+  url: https://github.com/paulrobertlloyd
+image:
+  src: /assets/posts/govuk-eleventy-plugin/govuk-eleventy-plugin.png
+  alt: The homepage illustration for the GOV.UK Eleventy Plugin.
+---
+
+Back in 2018, the Becoming a teacher team at the Department for Education wanted to [document the design evolution of the services they were building](https://design-history.herokuapp.com/keeping-a-design-history/). A small site was built using the [GOV.UK Prototype Kit](https://govuk-prototype-kit.herokuapp.com/docs), but this soon became difficult to manage as more people joined the team and wanted to write posts.
+
+Looking for alternatives, the team choose [Eleventy](https://11ty.dev), a simple static site generator developed by Zach Leatherman[^1]. Like the Prototype Kit, it uses Node.js, yet is designed around organising and publishing content-focused websites.
+
+This includes support for [Markdown](https://www.markdownguide.org), a lightweight markup language for formatting plain text documents. Posts can incorporate additional data like publication dates and authors, and be organised into different collections.
+
+With this foundation in place, [the design history](https://bat-design-history.netlify.app/) grew to over 500 posts written by more than 20 authors. In 2020 we made this tool available as [a template repository on GitHub](https://github.com/x-govuk/govuk-design-history) so that other teams could quickly begin writing their own design histories. The tool has since been adopted by other teams, both inside the DfE and across government.
```

Comment:
Suggest spelling out the department name in full, or defining the acronym earlier. Also I don’t think it’s common to use “the” with department acronyms?

```suggestion
With this foundation in place, [the design history](https://bat-design-history.netlify.app/) grew to over 500 posts written by more than 20 authors. In 2020 we made this tool available as [a template repository on GitHub](https://github.com/x-govuk/govuk-design-history) so that other teams could quickly begin writing their own design histories. The tool has since been adopted by other teams, both inside the Department for Education and across government.
```